Library/ViewModels/PledgeViewModel.swift

```diff
@@ -647,25 +634,6 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
           refTag: refTag
         )
       }
-
-    createBackingDataAndIsApplePay.takeWhen(createBackingCompletionEvents)
-      .observeValues { data, isApplePay in
-        let (properties, eventTags) = optimizelyTrackingAttributesAndEventTags(
-          with: data.project,
-          refTag: data.refTag
-        )
-
-        let allEventTags = eventTags
-          .withAllValuesFrom(optimizelyCheckoutEventTags(createBackingData: data, isApplePay: isApplePay))
-
-        try? AppEnvironment.current.optimizelyClient?
-          .track(
-            eventKey: "App Completed Checkout",
```

Comment:
I understood that we still wanted this event.

Reply:
Nope, the backend will be sending this.